includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -288,7 +289,9 @@ public function getData() {
 							'type' => 'translatable-text',
 							'key' => 'global-footer-site-overview-link-local-sitemap'
 						],
-						'href' => $this->getHref( 'local-sitemap' )
+						'href' => $this->getHref(
```

Comment:
please move condition to function. whole `$data` array is build like that.
you can create method `getLocalSitemapLink()` and call it there.